x-pack/filebeat/module/awsfargate/log/ingest/pipeline.yml

```diff
@@ -0,0 +1,15 @@
+description: "Pipeline for AWS Fargate logs"
+
+processors:
+  - set:
+      field: event.ingested
+      value: '{{_ingest.timestamp}}'
+  - rename:
+      field: message
+      target_field: event.original
+      ignore_missing: true
```

Comment:
Could you provide some detail on why doing this rename?

Reply:
Ahh sorry!! I thought we are moving all `message` to `event.oiriginal`. But I see https://github.com/elastic/beats/issues/14708 is the other way around. I will remove this renaming here.